src/tests/system/tests/test_netgroups.py

```diff
@@ -108,3 +109,145 @@ def test_netgroups__add_remove_netgroup_member(client: Client, provider: Generic
     assert len(result.members) == 1
     assert "(-, user-1)" not in result.members
     assert "(-, user-2)" in result.members
+
+
+@pytest.mark.parametrize("Operation", ["Add", "Replace"])
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__user_attribute_membernisnetgroup_uses_group_dn(
+    client: Client, provider: GenericProvider, Operation: str
+):
+    """
+    :title: User's 'memberNisNetgroup' attribute values are the DN of the group.
+    :setup:
+        1. Create users, groups.
+        2. Create a new netgroup called QAUsers and add a member (ng1) to QAUsers
+        3. Create another netgroup named DEVUsers and add a member (ng2) to DEVUsers
+        4. Modify the DEVUsers netgroup to replace its members with the members of QAUsers.
+        5. Start sssd
+    :steps:
+        1. Retrieve all members of the DEVUsers netgroup.
+        2. Confirm that the member directly added to DEVUsers is present.
+        3. Confirm that the member from QAUsers is now part of DEVUsers.
+    :expectedresults:
+        1. All members should be retrieved
+        2. Members directly added to DEVUsers is present.
+        3. Members from QAUsers is now part of DEVUsers.
+    :customerscenario: False
+    """
+    if isinstance(provider, IPA):
+        pytest.skip(reason="Not for IPA povider")
```

Comment:
It would be best to define
```
@pytest.mark.topology(KnownTopology.LDAP)
@pytest.mark.topology(KnownTopology.AD)
@pytest.mark.topology(KnownTopology.Samba)
```
and remove this from the test case

Reply:
Same for other test cases